tests/foreman/cli/test_computeresource_azurerm.py

```diff
@@ -528,3 +545,122 @@ def test_positive_azurerm_host_provisioned(
         # Azure cloud
         assert self.hostname.lower() == azureclient_host.name
         assert self.vm_size == azureclient_host.type
+
+
+@run_in_one_thread
+class TestAzureRm_BYOS_FinishTemplate_Provisioning:
+    """ AzureRM Host Provisioning Test with BYOS Image
+
+    """
+
+    @pytest.fixture(scope='class', autouse=True)
+    def class_setup(
+        self, request, module_domain, module_azurerm_cr, module_azurerm_byos_finishimg
+    ):
+        """
+        Sets Constants for all the Tests, fixtures which will be later used for assertions
+        """
+        request.cls.region = settings.azurerm.azure_region
+        request.cls.rhel7_ft_img = AZURERM_RHEL7_FT_BYOS_IMG_URN
```

Comment:
Why initializing them again in this class. We should directly use them as constants. It doesnt provides any advantage.

Reply:
Good catch, it's removed now.